operator/pkg/util/k8s.go

```diff
@@ -128,3 +130,23 @@ func CreateNamespace(cs kubernetes.Interface, namespace string, network string,
 
 	return nil
 }
+
+func PrometheusPathAndPort(pod *v1.Pod) (string, int, error) {
+	path := "/metrics"
+	port := 9090
```

Comment:
Why also change the default value?

Reply:
prometheus will use it, if you don't specify a specific path via `prometheus.io/path`.

Reply:
I see how it works. Thanks